assets/src/tinymce/components/Modal.js

```diff
@@ -18,6 +18,8 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import { __ } from '@web-stories-wp/i18n';
```

Comment:
Everything that is _not_ story editor or story dashboard should keep using `@wordpress/i18n`.

So need to undo that change here :-)

Reply:
Yep, as mentioned below, this change was warranted by the eslint, will check the eslint rules and see if we need to exclude this files.